### ISSUE TITLE
How do I gain access to the information sent to Stripe? 

### ISSUE THREAD

**Issue by omscsgt** [laurihy/angular-payments#76]
I'm using stripeForm which sends the data to stripe and provides the callback. But I would like to save the users last 4 digits of the credit card on my server. When the user enters the card numbers and puts submit, is there a way I can get the card number on client side and parse the last 4 digits out of it? It seems like all that would be inside

**Comment by pjarnfelt** [comment]
Put an ng-model on it and treat is as a normal form element. 
<input type="text" class="form-control" name="card_email" ng-model="card_email">
$scope.handleStripe = function(status, response){
    if(response.error) {
      alert("Error happened")
    } else {
      // got stripe token, now charge it 
      var dataModel = {
        token: response.id,
        email:$scope.card_email
      };

    }
  }

**Comment by bendrucker** [comment]
That's correct. And once you're reading those values into models anyway, you may want to look at https://github.com/bendrucker/angular-stripe instead. Instead of giving it a callback via a directive, you just call it from your form handler and keep everything in code. But instead of the normal Stripe.js behavior of using callbacks, you get a promise back.
